src/test/ui/higher-rank-trait-bounds/normalize-under-binder/normalization-under-binders-2.rs

```diff
@@ -0,0 +1,57 @@
+// ignore-compare-mode-nll
+// fails in migrate, passes in nll
```

Comment:
This is a bit weird, not sure

Reply:
I spent some time digging into this but I'm not sure what's going on yet.